docker/README.md

```diff
@@ -24,35 +28,54 @@ volumes:
 
 ### Use an Enrolled Identity from the Environment
 
-You may source an existing identity from the environment.
+You may source an existing identity from an environment variable. The value will be written to the mounted volume and the variable ignored thereafter. The identity file in the mounted volume will be overwritten when the certificate is automatically renewed.
```

Comment:
I started a thoughtful comment about rewriting this. As I was about 80% through it occurs to me that given the new need for a writable volume, all this feels like it should be changed to "put the identity file into the writable volume"

Reply:
We can deprecate the env var method before we decommission it, but I wouldn't support that since the utility and viability of the env var method hasn't changed. The alternative is substantially more complex and error prone.

Reply:
we can simply remove the doc as a first step. I think we should just remove it entirely. We can discuss the substantial more complexity in person, I don't see it...